barebones/core_portme.c

```diff
@@ -108,7 +114,7 @@ ee_u32 default_num_contexts=1;
 */
 void portable_init(core_portable *p, int *argc, char *argv[])
 {
-	#error "Call board initialization routines in portable init (if needed), in particular initialize UART!\n"
+//	#error "Call board initialization routines in portable init (if needed), in particular initialize UART!\n"
```

Comment:
I don't get this, can you explain?

Reply:
When core_main is run it calls the portable_init which is a generic function that the user can modify to set up their test hardware environment before running the test. The writers of the test sprinkled #errors throughout the code to make sure you saw and modified these sections since they could be critical to running the test (you can see another error inserted in the barebones clock section)

Reply:
Ah, makes sense. That’s kind of silly, but what are you gonna do :)